MediaRecorder.bs

```diff
@@ -410,6 +405,67 @@ interface MediaRecorder : EventTarget {
   </table>
   </dd>
 
+  <dt><dfn method for="MediaRecorder"><code>replaceTrack(MediaStreamTrack track, MediaStreamTrack withTrack)</code></dfn></dt>
+  <dd>
+  When a {{MediaRecorder}} object&#8217;s {{replaceTrack()}} method is invoked,
+  the UA MUST run the following steps:
+  <ol>
+    <li>Let |recorder| be the {{MediaRecorder}} object on which the method was
+    invoked.</li>
+    <li>Let |track| be the method's first argument.</li>
+    <li>Let |withTrack| be the method's second argument.</li>
+    <li>If |recorder|'s {{state}} is {{inactive}}, return a promise rejected
+    with a newly created {{InvalidStateError}} {{DOMException}} and abort these
+    steps.</li>
+    <li>If |track| is not found in |recorder|'s [=[[Tracks]]=], return a promise
+    rejected with a newly created {{NotFoundError}} {{DOMException}} and abort
+    these steps.</li>
+    <li>If |track| is {{ended}}, or |withTrack|'s {{kind}} differs from
+    |track|'s {{kind}}, then return a promise rejected with a newly created
+    {{InvalidModificationError}} {{DOMException}} and abort these steps.</li>
+    <li>If the <a>isolation properties</a> of |track| or |withTrack| disallow
+    access from |recorder|, return a promise rejected with a newly created
+    {{SecurityError}} {{DOMException}} and abort these steps.</li>
+    <li>If the User Agent cannot record |withTrack| in place of |track| using
+    the current configuration, then return a promise rejected with a newly
+    created {{NotSupportedError}} {{DOMException}} and abort these steps.</li>
+    <li>Let |session| be the ongoing recording session corresponding to the
```

Comment:
"recording session" needs a definition somewhere. There is no mention of "session" anywhere in the spec.

I can't really guess what it means either since intuitively a recorder cannot have more than one session at a time (it produces one output file across several blobs). But here it seems like the recorder has one session per track?